test/org/opensolaris/opengrok/condition/ConditionalRunRepeatable.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+ /*
+ * Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opensolaris.opengrok.condition;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Annotation allowing the {@link ConditionalRun} to be repeatable annotation.
+ *
+ * @author Krystof Tulinger
+ */
+@Retention(value = RUNTIME)
+@Documented
+@Target(value = {ElementType.METHOD, ElementType.TYPE})
+public @interface ConditionalRunRepeatable {
+
+    ConditionalRun[] value();
```

Comment:
I am slightly confused by the `value` used in this file. Does it have any special significance to name the array so ?

Reply:
Oh, this seems to be the rule for containing annotation type as per https://docs.oracle.com/javase/tutorial/java/annotations/repeating.html

Reply:
yes it must be like this